src/job-exporter/src/collector.py

```diff
@@ -101,7 +101,8 @@ def __init__(self):
                 "username",
                 "job_name",
                 "role_name",
-                "task_index"
+                "task_index",
+                "app_id",
```

Comment:
please use a more specific name, maybe `retry_count`.

Reply:
It's not a `retry_count`. It's a job instance id such as `application_1553664769226_0080`. This id will be different in each job retry.
I use `app_id` here because it's same with the `app_id` filed in PAI restAPI. If we want to support multi-platform (such as DLWS), I can rename it to `job_instance_uid`.

Reply:
I see.

Reply:
Maybe job_instance_id, or instance_id. And please add comment here.

Reply:
Why use `uid`? it usually means user id

Reply:
Means unique id. Will change to `job_instance_id` here.

Reply:
Actually I think `attempt_id` is better, but it might be messy with yarn's attempt.